app/javascript/courses/students/images/reviewed-empty.svg

```diff
@@ -0,0 +1 @@
8 6.22a116.11 116.11 0 0 0-2-14.46z" class="E"/><g fill="#5b4b4b"><path d="M83.06 165.56h-.12c-1.15-.35-1-2.44-1-2.52h.38c-.05.52 0 1.9.68 2.1 0 0 .1 0 .3-.25 1.3-1.55 3.86-8.44 3.6-9.7a1.14 1.14 0 0 0-1.47-1l-.1-.38a1.8 1.8 0 0 1 1.24.17 1.64 1.64 0 0 1 .71 1.13c.3 1.46-2.54 9-3.85 10.2a.64.64 0 0 1-.37.23z"/><path d="M82.37 164.9a.25.25 0 0 1-.12 0c-1.16-.35-1-2-.94-2.06h.4s-.18 1.4.66 1.64c0 0 .1 0 .3-.26 1.3-1.55 3.86-8.44 3.6-9.7a1.27 1.27 0 0 0-.54-.87 1.33 1.33 0 0 0-.93-.13l-.1-.37a1.69 1.69 0 0 1 1.23.16 1.66 1.66 0 0 1 .72 1.13c.3 1.47-2.54 9-3.85 10.22a.67.67 0 0 1-.42.24z"/><path d="M81.66 163.6a.22.22 0 0 1-.12 0 1.37 1.37 0 0 1-1.05-1.44h.38a1 1 0 0 0 .78 1s.1 0 .3-.25c1.3-1.55 3.86-8.44 3.6-9.7a1.14 1.14 0 0 0-1.47-1l-.1-.38a1.8 1.8 0 0 1 1.24.17 1.62 1.62 0 0 1 .72 1.13c.3 1.46-2.54 9-3.86 10.2a.65.65 0 0 1-.41.26zm2.1 2.6h-.12c-1.15-.34-1-2.43-.95-2.52h.4c-.06.52 0 1.9.67 2.1 0 0 .1 0 .3-.26 1.3-1.55 3.86-8.43 3.6-9.7a1.27 1.27 0 0 0-.54-.88 1.33 1.33 0 0 0-.93-.13l-.1-.37a1.69 1.69 0 0 1 1.24.17 1.64 1.64 0 0 1 .71 1.13c.3 1.46-2.54 9-3.85 10.2a.76.76 0 0 1-.42.23z"/></g><path d="M85.8 119.47l2.06-.67c1.3-3.2-4.7-18.38-4.7-18.38l-2.3 1.28z" class="C"/><path d="M86.34 115.17a.84.84 0 0 0 .15-1.48c-.55-.5-1 0-1.26.3a.63.63 0 0 0-.11.61c.15.6.95.64 1.22.56zm-1.64-4.97a.84.84 0 0 0 .15-1.48c-.55-.52-1 0-1.26.3a.62.62 0 0 0-.11.6c.15.6.95.58 1.22.57zM83 105a.83.83 0 0 0 .15-1.47c-.55-.52-1 0-1.26.3a.63.63 0 0 0-.12.6c.15.56.95.56 1.23.56zm26.6 29.08a.84.84 0 0 0 .15-1.48c-.55-.52-1 0-1.26.3a.63.63 0 0 0-.11.61c.15.6.95.57 1.22.56zm-6.7-.45a.84.84 0 0 0 .15-1.48c-.55-.5-1 0-1.26.3a.63.63 0 0 0-.11.61c.15.6.95.57 1.22.56zM93 181c-.36-.66.36-1.08 1.14-1.7-.8.23-1-.64-1-.64-.2-1.53 3-.43 3.38 1s0 2.88-1 2.84c-1.14-.16-.73-1.85-.73-1.85-.68.82-1.68.85-1.8.36zm-17.2 27.3c-.35-.66.37-1.08 1.15-1.7-.8.23-1-.63-1-.63-.2-1.54 3-.44 3.38 1s0 2.88-1 2.84c-1.14-.16-.73-1.85-.73-1.85-.6.85-1.6.88-1.8.35zm.66-20.1c-.36-.66.37-1.1 1.15-1.72a.84.84 0 0 1-1-.63c-.18-1.53 3-.44 3.4 1s0 2.88-1 2.85c-1.14-.16-.73-1.85-.73-1.85-.65.84-1.66.88-1.8.35z" class="B"/><path d="M140.17 129.3c-2.2 2.75-1.86 14.45-1.86 14.45l-7.9 6s1-5.92 2.13-8.43a87.89 87.89 0 0 0 2.78-8.73z" class="E"/><path d="M155.6 129.66a.84.84 0 0 1-.57-.25l.28-.27a.49.49 0 0 0 .29.14.45.45 0 0 0 .3-.17.71.71 0 0 0 .2-.58c-.06-1.48-3-4.13-4.55-5.55-.8-.76-1.27-.82-1.5-.74s-.45.7-.45.7l-.4-.05s.12-.8.7-1 1.13.1 1.9.82c2.23 2 4.62 4.3 4.68 5.82a1.14 1.14 0 0 1-.31.87.76.76 0 0 1-.58.27z" fill="#262323"/><path d="M117 73.44a1.77 1.77 0 0 0 1.31-2.92c-.85-1.3-2.16-.66-2.82-.17a1.78 1.78 0 0 0-.69 1.23c-.02 1.27 1.6 1.88 2.2 1.86zm.33-4.56a1.78 1.78 0 0 0 1.31-2.88c-.85-1.3-2.17-.66-2.82-.17a1.75 1.75 0 0 0-.69 1.22c-.03 1.3 1.6 1.85 2.2 1.83zm-.6 8.06a1.37 1.37 0 0 0 .93-2.18c-.64-1-1.56-.48-2.08-.13a1 1 0 0 0-.51.89c-.07.94 1.12 1.37 1.66 1.42z" fill="#fff"/><ellipse cx="264.17" cy="58.83" rx="3.2" ry="5.06" fill="#e2e8f0"/><ellipse cx="204.85" cy="192.79" rx="3.2" ry="5.06" class="F"/></svg>
```

Comment:
use the image from the reviewed submission pack instead of duplicating the file.

Reply:
Fixed in 0714ae32